.babelrc

```diff
@@ -1,5 +1,19 @@
 {
-  "presets": ["es2015"],
+  "presets": [
+    ["env", {
+      "debug": true,
+      "targets": {
+        "node": 4.7,
+        "browsers": [
```

Comment:
I am not sure if we should support browsers, content-import and export are mainly not intended to use in a browser env, I guess we could remove the `"browsers"` prop